docs/AGENT_INTEGRATION_GUIDE.md

```
@@ -68,28 +68,25 @@ docker exec ping-mem-neo4j cypher-shell -u neo4j -p neo4j_password "RETURN 1"  #
-        "QDRANT_URL": "http://localhost:6333",
-        "QDRANT_COLLECTION_NAME": "ping-mem-vectors",
-        "QDRANT_VECTOR_DIMENSIONS": "768"
+        "PING_MEM_REST_URL": "http://localhost:3003"
       }
     }
   }
 }
 ```
 
+> **Note**: Direct mode (`dist/mcp/cli.js`) is deprecated — it opens the DB directly and causes concurrent access issues with Docker. Use proxy mode with `PING_MEM_REST_URL` instead.
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)